dev-lisp/newlisp/newlisp-10.7.5.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="Put the fun back in Lisp"
+DESCRIPTION="newLISP is a LISP-like scripting language for doing things"
+HOMEPAGE="https://www.newlisp.org"
+COPYRIGHT="2013 Lutz Mueller"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.newlisp.org/downloads/newlisp-10.7.5.tgz"
+SOURCE_FILENAME="newlisp-$portVersion.tar.gz"
+CHECKSUM_SHA256="dc2d0ff651c2b275bc4af3af8ba59851a6fb6e1eaddc20ae75fb60b1e90126ec"
+PATCHES="newlisp-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
Maybe add commandSuffix etc here, atm on 32bit binaries are installed in $prefix/bin/x86 (where the cmd's in PROVIDES need $secondaryArchSuffix, however this doesn't build with gcc2 so we would like the binaries to be in $prefix/bin instead on secondary architecture.
Example: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#commandbindir-and-commandsuffix